### ISSUE TITLE
Sass v1.58.4 won't compile

### ISSUE THREAD

**Issue by csphilli** [sass/dart-sass#1909]
The last time I used sass was version 1.58.3 and it compiled fine. I upgraded to the latest version and it wouldn't compile my very simple project. I decided to go through each previous version to see where it starts having issues and it's 1.58.4. 

The initial error is 
``` error  in ./src/views/jobs/Jobs.vue?vue&type=style&index=0&id=62687b36&lang=scss&scoped=true
Syntax Error: Error: PostCSS received undefined instead of CSS string
```
I'm not directly using PostCSS so I'm assuming it's a dependency. The stack trace below references
```
Module Error (from ./node_modules/sass-loader/dist/cjs.js):
Unexpected token '?'
```
My `package.json`:
```
{
  "name": "jobs",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build"
  },
  "dependencies": {
    "core-js": "^3.8.3",
    "sass-loader": "^13.2.0",
    "sass": "^1.58.4",
    "vue": "^3.2.13",
    "vue-router": "^4.0.3"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "~5.0.0",
    "@vue/cli-plugin-router": "~5.0.0",
    "@vue/cli-service": "~5.0.0"
  }
}
```
I'm using very basic scss styling within the ` ./src/views/jobs/Jobs.vue` file and can't find anything syntactically wrong with it:
```
<style lang="scss" scoped>
.job {
	h2 {
		background: #f4f4f4;
		padding: 20px;
		border-radius: 10px;
		margin: 10px auto;
		max-width: 600px;
		cursor: pointer;
		color: #444;
	}

	h2:hover {
		background: #ddd;
	}

	a {
		text-decoration: none;
	}
}
</style>
```
Some trace:
```
ERROR in ./src/views/jobs/Jobs.vue?vue&type=style&index=0&id=62687b36&lang=scss&scoped=true (./node_modules/css-loader/dist/cjs.js??clonedRuleSet-22.use[1]!./node_modules/vue-loader/dist/stylePostLoader.js!./node_modules/postcss-loader/dist/cjs.js??clonedRuleSet-22.use[2]!./node_modules/sass-loader/dist/cjs.js??clonedRuleSet-22.use[3]!./node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./src/views/jobs/Jobs.vue?vue&type=style&index=0&id=62687b36&lang=scss&scoped=true)
Module Error (from ./node_modules/sass-loader/dist/cjs.js):
Unexpected token '?'
```
I don't know what the unexpected token is but it's not present in my styling.

I can downgrade to sass version 1.58.3 and it compiles just fine.

**Comment by yncj** [comment]
Metoo,but i used v1.59.2
<img width="440" alt="image" src="https://user-images.githubusercontent.com/24471470/224496925-6014c7f9-a749-4d64-bd95-7f8a4447748e.png">

**Comment by ntkme** [comment]
Nullish coalescing operator (??) is supported in node >=14.

Sass only targets to support up to 3 most recent node LTS versions, meaning the current support target is node 18, 16, 14. Any node version <14 is no longer a support target.

**Comment by csphilli** [comment]
Upgrading to the required node version resolved this for me

**Comment by kadinwu** [comment]
i met this issue too, and my nodejs version is 14.17.1

**Comment by TitusRobyK** [comment]
try updating the node version to 14.17.6

**Comment by mhmtonrn** [comment]
> try updating the node version to 14.17.6

👍

**Comment by jathak** [comment]
We inadvertently started using JS features that were unsupported in Node 12 without declaring we were dropping support  in our package.json. We've now published a new version (1.59.3) that should be compatible with Node 12 again.

While we do plan to drop support for Node 12 soon, since it's EOL, we shouldn't have done so without updating the package.json, and we're sorry for the trouble.
